src/components/app/filter-dropdown.jsx

```diff
@@ -334,30 +337,26 @@ const FilterDropdown = React.createClass({
       </BS.NavDropdown>
     );
   }
+}
 
-});
+function FilterDropdownShell(props) {
+  const {repoInfos} = props;
 
+  if (repoInfos.length) {
+    const [{repoOwner, repoName}] = repoInfos;
 
-const FilterDropdownShell = React.createClass({
-  render() {
-    const {repoInfos} = this.props;
-
-    if (repoInfos.length) {
-      const [{repoOwner, repoName}] = repoInfos;
-
-      // TODO: Poll all of these so we get updates
```

Comment:
Create a separate PR (to be merged first) remove any of the TODOs which are going to be needlessly moved as part of this PR.  https://github.com/coala/gh-board/issues/30

Reply:
Not done yet?